tests/integration/test_logcollector/test_macos/test_macos_file_status_basic.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+import re
+import pytest
+import wazuh_testing.logcollector as logcollector
+import wazuh_testing.tools.macos_log.macos_utils as macos_utils
```

Comment:
Update if changes in macos_utils location are applied.
```suggestion
import wazuh_testing.tools.macos_log.macos_utils as macos_utils
```